client/client.go

```diff
@@ -1719,3 +1726,47 @@ func trimHTTPPrefix(str string) string {
 	str = strings.TrimPrefix(str, "https://")
 	return str
 }
+
+func (c *client) LoadGlobalConfig(ctx context.Context, names []string) ([]string, error) {
+	r, err := c.getClient().LoadGlobalConfig(ctx, &pdpb.LoadGlobalConfigRequest{Name: names})
+	if err != nil {
+		return nil, err
+	}
+	return r.Values, nil
+}
+
+func (c *client) StoreGlobalConfig(ctx context.Context, name string, value string) error {
+	_, err := c.getClient().StoreGlobalConfig(ctx, &pdpb.StoreGlobalConfigRequest{Name: name, Value: value})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *client) WatchGlobalConfig(ctx context.Context, receiver chan []string) error {
+	r, err := c.getClient().WatchGlobalConfig(ctx, &pdpb.WatchGlobalConfigRequest{})
+	if err != nil {
+		return err
+	}
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		defer func() {
+			wg.Done()
+		}()
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			default:
+			}
+			m, err := r.Recv()
+			if err != nil {
+				return
+			}
+			receiver <- []string{m.Name, m.Value}
+		}
+	}()
+	wg.Wait()
+	return err
+}
```

Comment:
I don't get the proper way to use this method.

1. It seems that it will block the caller forever until we cancel the `ctx`. Maybe a timeout detection is needed.
2. The `err` that `r.Recv()` return will never be known by the caller or developer. At least a log is needed.
3. This `for` loop doesn't have any waiting mechanism, I am worried about it will consume too much CPU usage.

Reply:
fixed